providers/ms365/resources/ms365.lr

```diff
@@ -85,6 +85,8 @@ microsoft.users {
 microsoft.conditionalAccess {
   // Named locations container
   namedLocations microsoft.conditionalAccess.namedLocations
+  // Policies collection
+  policies microsoft.conditionalAccess.policies
```

Comment:
why not directly return `[]microsoft.conditionalAccess.policy` here? Generally, we create these "wrapper" resources when we have extra functions or properties defined for the list. In your case you just return a list so you don't need the extra abstraction

Reply:
I think what you want is to have
```
policies() []microsoft.conditionalAccess.policy
```

This will define a function that returns a list and you won't have an extra layer of needless abstraction